clients/go/test/util/public_key_server.go

```diff
@@ -19,29 +19,46 @@ import (
 	"encoding/json"
 	"encoding/pem"
 	"fmt"
+	"io"
 	"log"
 	"net/http"
 	"net/http/httptest"
+	"os"
 	"strings"
 
 	"github.com/lestrrat-go/jwx/v2/jwk"
 )
 
-// Matching private key here: https://github.com/abcxyz/lumberjack/blob/92782c326681157221df37e0897ba234c5a22240/integration/testrunner/grpcrunner/grpc.go#L60
-const publicKeyString = `
------BEGIN PUBLIC KEY-----
-MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEhBWj8vw5LkPRWbCr45k0cOarIcWg
-ApM03mSYF911de5q1wGOL7R9N8pC7jo2xbS+i1wGsMiz+AWnhhZIQcNTKg==
------END PUBLIC KEY-----
-`
-
 // StartLocalPublicKeyServer parse pre-made key and set up a server to host it in JWKS format.
 // This is intended to stand in for the JVS in the integration tests.
+type publicKeyJSONData struct {
+	Encoded string
+	Decoded string
+}
+
+func loadJSON() (publicKeyJSONData, error) {
+	var data publicKeyJSONData
+	jsonFile, err := os.Open("/etc/lumberjack/public_key.json")
+	if err != nil {
+		return data, fmt.Errorf("failed opening files %w", err)
+	}
+	byteValue, _ := io.ReadAll(jsonFile)
+	err = json.Unmarshal(byteValue, &data)
+	if err != nil {
+		return data, fmt.Errorf("failed parsing files: %w", err)
```

Comment:
Same above, What is the empty value of "data", should we just return the empty value?

Reply:
Maybe use "nil" when we change the return to pointer?